packages/dd-trace/test/profiling/config.spec.js

```diff
@@ -133,7 +133,7 @@ describe('config', () => {
   it('should support profiler config with DD_PROFILING_PROFILERS', () => {
     process.env = {
       DD_PROFILING_PROFILERS: 'wall',
-      DD_PROFILING_EXPERIMENTAL_CODEHOTSPOTS_ENABLED: '1',
```

Comment:
Perhaps add a test that both options are supported and that DD_PROFILING_CODEHOTSPOTS_ENABLED overrides DD_PROFILING_EXPERIMENTAL_CODEHOTSPOTS_ENABLED

Reply:
Done.

Reply:
Thanks for the changes !